Moderation-Policy.md

```diff
@@ -115,8 +114,9 @@ a Post from Moderation.
 ## Guidelines and Requirements
 
 * All Posts are expected to respect the Node.js [Code of Conduct][].
-* Any Collaborator with commit rights to a given repository may Moderate Posts
-  within that repository's issue tracker.
+* Any Collaborator with triage permission to a given repository (except
+  nodejs/moderation) may Moderate Posts within that repository's issue tracker.
+  Only the Moderation team is allowed to moderate posts on nodejs/moderation.
```

Comment:
Is it intentional to exclude TSC voting members from moderating posts in the moderation repo? I'm not opposed, necessarily, but also want to make sure that's intentional.

Reply:
IIUC, the current moderation policy does not grant TSC voting members (nor Moderation Team members) any special privilege for moderating. This PR does not change that. I guess we could add `or the TSC`, but I can't think of any scenario where the Moderation Team would not want to moderate a post but the TSC would.

Reply:
Hmm, the TSC are org owners and have permission to moderate in any repo as if they have triage/write permission (which they do) as far as I understand?

Reply:
That's correct. As org owners, the TSC voting members (and Moderation Team members) have triage permissions on all repos in the org, and therefore the fit the "Collaborator" category for all repos in the org – i.e. they can moderate posts from non-collaborators without consulting the moderation team (although the Moderation Team should be notified, unless it's clear the person is not contributing in good faith), and from collaborators if the Moderation Team agrees.